Maple2.Model/Game/Sync/StateSync.cs

```diff
@@ -36,8 +36,8 @@ public enum Flag : byte {
     public int SyncNumber;
 
     #region Flag1
-    public int Flag1Unknown1;
-    public short Flag1Unknown2;
+    public int Flag1EmotionId; // Skill ID for emotes + flying mounts (not ground)
```

Comment:
Should we rename just to EmotionId?

I guess "Flag1" was used when we didn't knew what was about it

Reply:
I kept the Flag1 in there as some of the other variables were named similarly like "Flag4Animation" and "Flag6Position" but I can change it if we're going with dropping the flag part

Reply:
@Zintixx should we leave "Flag1" or nah?

Reply:
I think for now it's fine. We still don't _really_ know what this entails and will need more analyzing